tests/run-pass/backtrace-api.stderr

```diff
@@ -3,16 +3,16 @@ $DIR/backtrace-api.rs:12:50 (func_c)
 $DIR/backtrace-api.rs:6:53 (func_b)
 $DIR/backtrace-api.rs:5:50 (func_a)
 $DIR/backtrace-api.rs:17:18 (main)
-RUSTLIB/core/src/ops/function.rs:LL:COL (<fn() as std::ops::FnOnce<()>>::call_once - shim(fn()))
-RUSTLIB/std/src/sys_common/backtrace.rs:LL:COL (std::sys_common::backtrace::__rust_begin_short_backtrace)
-RUSTLIB/std/src/rt.rs:LL:COL (std::rt::lang_start::{closure#0})
-RUSTLIB/core/src/ops/function.rs:LL:COL (std::ops::function::impls::call_once)
-RUSTLIB/std/src/panicking.rs:LL:COL (std::panicking::r#try::do_call)
-RUSTLIB/std/src/panicking.rs:LL:COL (std::panicking::r#try)
-RUSTLIB/std/src/panic.rs:LL:COL (std::panic::catch_unwind)
-RUSTLIB/std/src/rt.rs:LL:COL (std::rt::lang_start_internal::{closure#2})
-RUSTLIB/std/src/panicking.rs:LL:COL (std::panicking::r#try::do_call)
-RUSTLIB/std/src/panicking.rs:LL:COL (std::panicking::r#try)
-RUSTLIB/std/src/panic.rs:LL:COL (std::panic::catch_unwind)
-RUSTLIB/std/src/rt.rs:LL:COL (std::rt::lang_start_internal)
-RUSTLIB/std/src/rt.rs:LL:COL (std::rt::lang_start)
+rustc_src/src/ops/function.rs:227:5 (<fn() as std::ops::FnOnce<()>>::call_once - shim(fn()))
```

Comment:
Looks like this line also needs to be adjusted to avoid leaking the line numbers:
https://github.com/rust-lang/miri/blob/5e3be823b2d97a0e7411ee34bd02df38f5f35b92/tests/run-pass/backtrace-api.rs#L2